angularjs-portal-frame/src/main/webapp/partials/sidebar-left.html

```diff
@@ -1,6 +1,13 @@
 <div>
   <nav class="side-bar-nav">
-    <span class="fa fa-arrow-left hide-sidebar" ng-click="$storage.showSidebar = false" popover="Minimize sidebar" popover-placement="left" popover-trigger="mouseenter" popover-popup-delay="600"></span>
+    <div class="hide-sidebar-div">
+      <span class="fa fa-arrow-left hide-sidebar" ng-click="$storage.showSidebar = false" popover="Minimize sidebar" popover-placement="left" popover-trigger="mouseenter" popover-popup-delay="600"></span>
+    </div>
+    <div class="profile-pic">
+      <img src="img/jared.jpeg">
```

Comment:
Is the file `jared.jpeg` meant to be in this pull request?  I don't think it's in [the `img` directory](https://github.com/UW-Madison-DoIT/angularjs-portal/tree/MUMUP-1437/angularjs-portal-frame/src/main/webapp/img)?